services/search/search_engine.py

```diff
@@ -71,7 +74,7 @@ def _calculate_score(self, frequency, weight, words, target_dict):
                     if score != 0:
                         target_dict[key] = score
 
-    def _search_query(self, keywords, synonyms):
+    def search_query(self, keywords, synonyms):
```

Comment:
Is this change related to the error exception somehow? It's not clear why it was included.

Reply:
This is not related at all, it is because pytest was not finding the method and since this was a small pr I decided to add it here.